lib/Transforms/ConvertIfToSelect/BUILD

```
@@ -0,0 +1,43 @@
+load("@llvm-project//mlir:tblgen.bzl", "gentbl_cc_library")
+
+package(
+    default_applicable_licenses = ["@heir//:license"],
+    default_visibility = ["//visibility:public"],
+)
+
+cc_library(
+    name = "ConvertIfToSelect",
+    srcs = ["ConvertIfToSelect.cpp"],
+    hdrs = ["ConvertIfToSelect.h"],
+    deps = [
+        ":pass_inc_gen",
+        "@llvm-project//mlir:IR",
+        "@llvm-project//mlir:Pass",
+        "@llvm-project//mlir:SCFDialect",
```

It probably gets the Arith dependency transivitely from SCF or something else already, but I think the pass pass should explicitly depend on ArithDialect

```suggestion
        "@llvm-project//mlir:SCFDialect",
        "@llvm-project//mlir:ArithDialect",
```

---

Google has an internal rule that says something like "all included files require direct dependencies exporting those files," but fixing this is part of our internal process for handling contributed PRs. Long story short, if you don't do this then we'll do it automatically for you :)